governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Onboard/Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022
+summary: This is a budget amendment application for Keep3r Network: a decentralized and permissionless keeper network.
```

Comment:
Not sure if these fields can contain a colon. @prose11 ?

Reply:
this is why I had to change the file title...

Reply:
For reference, the "|" was also breaking it, I've changed it to "Onboard-Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md"